engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/MainMenuScreen.java

```diff
@@ -56,7 +56,7 @@ public void initialise() {
         WidgetUtil.trySubscribe(this, "settings", button -> triggerForwardAnimation(SettingsMenuScreen.ASSET_URI));
         WidgetUtil.trySubscribe(this, "credits", button -> triggerForwardAnimation(CreditsScreen.ASSET_URI));
         WidgetUtil.trySubscribe(this, "exit", button -> engine.shutdown());
-        WidgetUtil.trySubscribe(this, "crashReporter", widget -> CrashReporter.report(new Throwable("Report an error."), LoggingContext.getLoggingPath()));
+        WidgetUtil.trySubscribe(this, "crashReporter", widget -> CrashReporter.report(new Throwable("Report a error."), LoggingContext.getLoggingPath(), false));
```

Comment:
Hey, I'm just thinking about deleting all the text "Report a error" or changing it, because there is actually no error in issue reporter.